backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -73,7 +73,7 @@ async function calculateAndAddDecayTransactions(
     finalTransaction.totalBalance = roundFloorFrom4(userTransaction.balance)
     const prev = i > 0 ? userTransactions[i - 1] : null
 
-    if (prev && prev.balance > 0) {
+    if (prev) {
```

Comment:
what does prev represent? previousTransaction?
So if previousTransaction is not null?